From d6d8fdf1423b7e50c71f51d9d36e9f852c9c72ce Mon Sep 17 00:00:00 2001
From: Wtz_LASR <wtz_lasr@outlook.com>
Date: Tue, 10 May 2022 03:30:57 +0800
Subject: [PATCH] fix luaL_loadbufferx redefinition

---
 single/sol/sol.hpp | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/single/sol/sol.hpp b/single/sol/sol.hpp
index 049a4c1f..b5378f99 100644
--- a/single/sol/sol.hpp
+++ b/single/sol/sol.hpp
@@ -1238,12 +1238,12 @@ inline const char* kepler_lua_compat_get_string(lua_State* L, void* ud, size_t*
 
 #if !defined(SOL_LUAJIT) || (SOL_LUAJIT_VERSION < 20100)
 
-inline int luaL_loadbufferx(lua_State* L, const char* buff, size_t size, const char* name, const char*) {
-    kepler_lua_compat_get_string_view ls;
-    ls.s = buff;
-    ls.size = size;
-    return lua_load(L, kepler_lua_compat_get_string, &ls, name/*, mode*/);
-}
+// inline int luaL_loadbufferx(lua_State* L, const char* buff, size_t size, const char* name, const char*) {
+//     kepler_lua_compat_get_string_view ls;
+//     ls.s = buff;
+//     ls.size = size;
+//     return lua_load(L, kepler_lua_compat_get_string, &ls, name/*, mode*/);
+// }
 
 #endif // LuaJIT 2.1.x beta and beyond
 
-- 
2.36.0.windows.1

